cmake.html.markdown

```diff
@@ -0,0 +1,176 @@
+---
+language: cmake
+contributors:
+    - ["Bruno Alano", "https://github.com/brunoalano"]
+filename: CMake
+---
+
+CMake it's a cross-platform, open-source build system. This tool will allow you
+to test, compile and create packages of your source code.
+
+The problem that CMake tries to solve it's the problem of Makefiles and
+Autoconfigure on cross-platform (different make interpreters have different
+command) and the ease-of-use on linking 3rd party libraries.
+
+CMake is an extensible, open-source system that manages the build process in
+an operating system and in a compiler-independent manner. Unlike many
```

Comment:
I would say, "CMake is an extensible, open-source system that manages the build process in
an operating-system- and compiler-independent manner."